src/api.rs

```diff
@@ -240,6 +242,26 @@ impl RapiClient for RapiReqwestClient {
             form = form.text("name", name)
         }
 
+        if let Some(env_args) = env_args {
+            for env_arg in env_args {
+                let key_value: Vec<&str> = env_arg.splitn(2, '=').collect();
+                if key_value.len() == 2 {
+                    let key = key_value[0];
+                    let value = key_value
+                        .get(1)
+                        .map(|val| val.to_string())
+                        .unwrap_or_else(|| "".to_string());
+                    if value.is_empty() {
+                        Err(EnvArgError::MissingValue { env_arg: env_arg.clone() })?
```

Comment:
This is already an Err instance, I suggest to just return it. Technically `?` works here but it doesn't really make sense since we know this is an error and we should return here

Reply:
254 | /                     if value.is_empty() {
255 | |                         Err(EnvArgError::MissingValue { env_arg: env_arg.clone() })
    | |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `()`, found `Result<_, EnvArgError>`